src/modules/drawer.scss

```diff
@@ -20,6 +20,9 @@ html:not(.ua-ie-edge):not(.ua-ie-11) body {
         width: 100%;
         z-index: $zindex + 2;
     }
+    &.drawer-sticky-mode{
+        position: unset !important; // sass-lint:disable-line no-important
```

Comment:
Is this the rule that's causing the page to jump back to the top of the viewport when opening up the sticky drawer?

Reply:
Yes, Because of it, Page is jumping back.

Reply:
Fixed. It took me sometime to figure out problem and fixed it. Now, its completed.